devices/Mcp7940xx/README.md

```diff
@@ -0,0 +1,75 @@
+# Mcp7940xx - I2C Real-Time Clock/Calendar with SRAM
+
+The MCP7940M Real-Time Clock/Calendar (RTCC) tracks time using internal counters for hours, minutes, seconds, days, months, years, and day of week. Alarms can be configured on all counters up to and including months.
+
+## Documentation
+
+[Datasheet](https://ww1.microchip.com/downloads/en/DeviceDoc/MCP7940M-Low-Cost%20I2C-RTCC-with-SRAM-20002292C.pdf)
+
+Original code was written for ESP32
+
+## Usage
+
+```csharp
+using System;
+using System.Device.I2c;
+using System.Diagnostics;
+using System.Threading;
+using Iot.Device.Mcp7940xx;
+using nanoFramework.Hardware.Esp32;
+
+// Setup ESP32 I2C port.
```

Comment:
I know you've documented this in the code but we have a typical paragraph that you can copy/paste before the code which explain that the pins need to be setup for ESP32 or selected properly for the other platform. It would be perfect to add this before the code.

Reply:
Done.